### PR TITLE
Second 1200+ Words Commit for Appendix C-Git-Commands

### PR DESCRIPTION
**Pull request by cherryloua**
Hello,

Please, take some time for review my second PR Request. The translation covers from `git difftool` command to fifth paragraph of `git log` command. 

Thanks,
cherryloua

### CODE REVIEW

**Comment by severinolorillajr** [comment]
Hi @cherryloua, please find someone from our group who can review and proofread your work.
Then let me know as soon as you have this reviewed. 

Thanks.

**Comment by mind-blast** [comment]
@cherryloua checking your translation. it seems @severinolorillajr is busy right now.. wait a moment.

File: book/C-git-commands/1-git-commands.asc [hunk 1]
```diff
@@ -102,8 +102,9 @@ Una nating na pag-usapang ang `status` sa << _ git_basics_chapter # _checking_st
 Habang ginagamit namin ito sa buong libro, halos lahat ng iyong magagawa sa `git status` na utos ay sakop doon.
 
 ==== git diff
+
 Ang kautosang `git diff` ay ginagamit kapag nais mong makita ang mga pagkakaiba sa pagitan ng dalawang trees.
-Ito ay maaaring pagkakaiba sa pagitan ng iyong gumaganang kaligiran at ang iyong staging na lugar (`git diff` mismo), sa pagitan ng iyong staging na lugar at ang iyong huling commit (` git diff --staged`), o sa pagitan ng dalawang mga commit (`git diff master branchB `).
+Ito ay maaaring pagkakaiba sa pagitan ng iyong gumaganang kaligiran at ang iyong staging na lugar (`git diff` mismo), sa pagitan ng iyong staging na lugar at ang iyong huling commit (` git diyff --staged`), o sa pagitan ng dalawang mga commit (`git diff master branchB `).
```
Comment (mind-blast) [inline book/C-git-commands/1-git-commands.asc]: (` git diyff --staged`)   please change to (` git diff --staged`)

File: book/C-git-commands/1-git-commands.asc [hunk 2]
```diff
@@ -118,64 +118,64 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
+Unang tinalakay natin ang mga pangunahing kaalaman sa pag-commit sa <<_git_basics_chapter#_committing_changes>>. Ipinakita din namin kung paano gamitin ang isang `-a` na palatandaan upang laktawan ang `git add` na hakbang sa pang-araw-araw na daloy ng trabaho at kung paano gamitin ang `-m` na palatandaan upang pumasa ng isang mensaheng commit sa command line sa halip ng pagpapatakbo nito sa editor.
 
-Sa _git_basics_chapter.asc tinatakpan namin ang paggamit ng --amam na opsyon upang gawing muli ang pinakabagong gawa.
+Sa _git_basics_chapter.asc tinalakay namin ang paggamit ng --amend na opsyon upang gawing muli ang pinakabagong commit.
 
-Sa _git_branching.asc, pumunta kami sa mas maraming detalye tungkol sa kung ano ang ginagawa ng git at kung bakit ginagawa nito ito.
+Sa _git_branching.asc, pumunta kami sa mas maraming detalye tungkol sa kung ano ang ginagawa ng git at kung bakit ganito niya ginagawa.
 
-Tiningnan namin kung paano mag-sign magawa ng cryptographically kasama ang `-S` flag sa <<_git_tools#_signing_commits>>.
+Tiningnan namin kung paano mag-sign ng mga commit na cryptographically kasama ang `-S` na palatandaan sa <<_git_tools#_signing_commits>>.
```
Comment (mind-blast) [inline book/C-git-commands/1-git-commands.asc]: i think flag is a reserve word on  git.  please check   :    flag    and    na palatandaan

File: book/C-git-commands/1-git-commands.asc [hunk 3]
```diff
@@ -118,64 +118,64 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
+=== Branching at Merging
 
-Mayroong isang maliit na bilang ng mga utos na nagpapatupad ng karamihan sa pagsasanib at pagsasama ng pag-andar sa Git.
+Iilan lamang ang bilang ng mga utos na karamihang nagpapatupad sa pagsasanib at pagsasama sa Git.
 
 ==== git branch
 
-Ang `git branch` command ay talagang isang bagay ng isang tool sa pamamahala ng sangay.
-Maaari itong ilista ang mga sangay na mayroon ka, lumikha ng isang bagong sangay, tanggalin ang mga sanga at palitan ang pangalan ng mga sanga.
+Ang `git branch` command ay talagang isang tool na naayon sa pamamahala ng sangay.
```
Comment (mind-blast) [inline book/C-git-commands/1-git-commands.asc]: please also review also.  branch is a reserve word. please recheck.

File: book/C-git-commands/1-git-commands.asc [hunk 4]
```diff
@@ -206,11 +206,11 @@ Susubukan nito ang kasalukuyang sangay sa resulta ng pagsasama.
...
 
-Tinakpan namin kung paano gumawa ng isang nakakulong na pagsasama (kung saan ang Git ay nagpahiwatig ng trabaho ngunit nagpapanggap na tulad ng isang bagong komit na hindi naitala ang kasaysayan ng sangay na pinagsasama mo) sa dulo ng << _ distributed_git # _public_project >>.
+Tinalakay namin kung paano gumawa ng isang squashed merge (kung saan ang Git ay nagpahiwatig na gumagana ang merge ngunit nagpapanggap lamang ito na tulad ng isang bagong commit na hindi naitala ang kasaysayan ng sangay na pinagsasama mo) sa pinakadulo ng << _ distributed_git # _public_project >>.
 
-Nagpunta kami ng maraming tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` command at ang` --abort` flag upang i-abort ang problema sa pagsasama sa << _ git_tools # _advanced_merging >>.
+Maramin din kaming tinalakay tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` na ustos at ang` --abort` na palatandaan upang i-abort ang problema sa pag-merge sa << _ git_tools # _advanced_merging >>.
```
Comment (mind-blast) [inline book/C-git-commands/1-git-commands.asc]: please change Maramin   to Marami  .   na ustos  to na utos

File: book/C-git-commands/1-git-commands.asc [hunk 5]
```diff
@@ -222,19 +222,17 @@ Binanggit namin ito nang mabilis sa << _ git_branching # _basic_merge_conflicts
...
 
-In <<_git_branching#_create_new_branch>> we use it with the `--decorate` option to easily visualize where our branch pointers are located and we also use the `--graph` option to see what divergent histories look like.
+Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorate` na opsyon para madaling mailarawan kung saan ang ating mga branch pointer ay nakalagay at ginamit din namin ang `--graph` na opsyon para makita kung ano ang hitsura ng mga palayong kasaysayan.
 
-In <<_distributed_git#_private_team>> and <<_git_tools#_commit_ranges>> we cover the `branchA..branchB` syntax to use the `git log` command to see what commits are unique to a branch relative to another branch.
-In <<_git_tools#_commit_ranges>> we go through this fairly extensively.
+Sa <<_distributed_git#_private_team>> at <<_git_tools#_commit_ranges>> aming tinalakay ang `branchA..branchB` na syntax para gamitin ang `git log` na utos para makita kung ano ang kaibahan sa isang katugong sangay sa iba pang sangay.
```
Comment (mind-blast) [inline book/C-git-commands/1-git-commands.asc]: change aming   to aming .    sangay   to  branch ..

**Comment by mind-blast** [comment]
Since i cannot edit it directly , please changed them according to what i suggest.  Thank you.

File: book/C-git-commands/1-git-commands.asc [hunk 6]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
+Ang `git difftool` na kautosan ay naglulunsad lamang ng isang panlabas na kasangkapan upang ipakita sa iyo ang pagkakaiba sa pagitan ng dalawang mga tree kung sakaling gusto mong gumamit ng iba pang bagay kaysa sa built in na `git diff` na kautusan.
 
-Ang git difftool command ay naglulunsad lamang ng isang panlabas na tool upang ipakita sa iyo ang pagkakaiba sa pagitan ng dalawang puno kung sakaling gusto mong gumamit ng iba pang bagay kaysa sa built in na git diff command.
-Binanggit lang namin ito sa _git_basics_chapter.asc.
+Binanggit lang namin ito sa <<_git_basics_chapter#_git_diff_staged>>.
 
 ==== git commit
 
-Ang `git commit` na kautusan ay kumukuha sa lahat ng mga nilalaman ng file na itinanghal sa `git add` at nagtatala ng isang bagong permanenteng snapshot sa database at pagkatapos ay inililipat ang branch pointer sa kasalukuyang branch hanggang dito.
+Ang `git commit` na kautusan ay kumukuha sa lahat ng mga nilalaman ng file na itinanghal sa `git add` at nagtatala ng isang bagong permanenteng snapshot sa database at pagkatapos ay inililipat ang branch pointer hangang sa kasalukuyang branch nito.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: hangang to hanggang?

File: book/C-git-commands/1-git-commands.asc [hunk 7]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
+Ang utos na `git reset` ay pangunahing ginagamit upang i-undo ang mga bagay, parang maaari mong sabihin sa pamamagitan ng pandiwa. Ito ay gumagalaw sa paligid ng HEAD pointer at opsyonal na nagbabago sa index o sa lugar ng staging at maaari ring opsyonal na baguhin ang gumaganang direktoryo kung gagamitin mo ang `--hard`. Ang pangwakas na opsyon ay posibleng magbura sa iyong tinatrabaho kung ginamit nang hindi tama, kaya tiyaking maunawaan mo ito bago gamitin.
 
-We first effectively cover the simplest use of `git reset` in <<_git_basics_chapter#_unstaging>>, where we use it to unstage a file we had run `git add` on.
-Una naming epektibong sinasakop ang pinakasimpleng paggamit ng git reset sa _git_basics_chapter.asc, kung saan ginagamit namin ito upang unstage isang file na kami ay tumakbo git idagdag sa.
+Una naming epektibong tinalakay ang pinakasimpleng paggamit ng git reset sa _git_basics_chapter.asc, kung saan ginagamit namin ito upang ma unstage isang file na pinatakbo ang `git add`.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: Please be careful with this tags <<_git_basics_chapter#_unstaging>>
Don't remove them.

File: book/C-git-commands/1-git-commands.asc [hunk 8]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
 
-We then cover it in quite some detail in <<_git_tools#_git_reset>>, which is entirely devoted to explaining this command.
-Pagkatapos ay tinatakpan namin ito sa lubos na detalye sa _git_tools.asc, na lubos na nakatuon sa pagpapaliwanag sa utos na ito.
+Pagkatapos ay tinalakay namin ito ng lubos ang detalye sa _git_tools.asc, na nakalaan sa pagpapaliwanag sa utos na ito.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: *namin to natin.
natin ito ng lubusan sa 
<<_git_tools#_git_reset>> Again please don't remove the tags.

File: book/C-git-commands/1-git-commands.asc [hunk 9]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
 
-We use `git reset --hard` to abort a merge in <<_git_tools#_abort_merge>>, where we also use `git merge --abort`, which is a bit of a wrapper for the `git reset` command.
-Ginagamit namin ang git reset - hard upang i-abort ang isang sumanib sa _git_tools.asc, kung saan ginagamit din namin ang git merge --abort, na kung saan ay isang bit ng wrapper para sa git reset command.
+Ginagamit namin ang git reset `--hard` upang i-abort ang pagsanib sa _git_tools.asc, kung saan ginagamit din namin ang `git merge --abort`, na kung saan ay isang kaunting pagbabalut para sa `git reset` command.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: <<_git_tools#_abort_merge>>

File: book/C-git-commands/1-git-commands.asc [hunk 10]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
-Maaari itong ilista ang mga sangay na mayroon ka, lumikha ng isang bagong sangay, tanggalin ang mga sanga at palitan ang pangalan ng mga sanga.
+Ang `git branch` command ay talagang isang tool na naayon sa pamamahala ng sangay.
+Maaari nitong ilista ang mga sangay na mayroon ka, lumikha ng isang bagong sangay, tanggalin ang mga sangay at palitan ang pangalan ng mga sangay.
 
-Most of <<_git_branching#_git_branching>> is dedicated to the `branch` command and it's used throughout the entire chapter.
-We first introduce it in <<_git_branching#_create_new_branch>> and we go through most of its other features (listing and deleting) in <<_git_branching#_branch_management>>.
-Karamihan ng << _ git_branching # _git_branching >> ay nakatuon sa command na 'branch` at ginagamit ito sa buong kabanata.
-Una naming ipakilala ito sa << _ git_branching # _create_new_branch >> at kami ay dumadaan sa karamihan ng iba pang mga tampok nito (listahan at pagtanggal) sa << _ git_branching # _branch_management >>.
+Karamihan ng << _ git_branching # _git_branching >> ay nakatuon sa utos na 'branch` at ginagamit ito sa buong kabanata.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: <<_git_branching#_git_branching>>

File: book/C-git-commands/1-git-commands.asc [hunk 11]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
+Una naming ipinakilala ito sa << _ git_branching # _create_new_branch >> at kami ay dumaan sa karamihan ng iba pang mga tampok nito (listahan at pagtanggal) sa << _ git_branching # _branch_management >>.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: << _ git_branching # _create_new_branch >>

File: book/C-git-commands/1-git-commands.asc [hunk 12]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
-We use it to reintroduce file conflicts with `--conflict=diff3` in <<_git_tools#_checking_out_conflicts>>.
-Ginagamit namin ito upang maipakita muli ang mga kontrahan ng file sa `--conflict = diff3` sa << _ git_tools # _checking_out_conflicts >>.
+Ginagamit namin ito upang maipakita muli ang mga magkasalungat na file sa `--conflict = diff3` sa << _ git_tools # _checking_out_conflicts >>.
 
-We go into closer detail on its relationship with `git reset` in <<_git_tools#_git_reset>>.
 Pumunta kami sa mas detalyadong detalye sa kaugnayan nito sa `git reset` sa << _ git_tools # _git_reset >>.
 
-Finally, we go into some implementation detail in <<_git_internals#_the_head>>.
-Sa wakas, pumunta kami sa ilang detalye ng pagpapatupad sa << _ git_internals # _the_head >>.
+Sa panghuli, pumunta kami sa ilang detalye ng pagpapatupad sa << _ git_internals # _the_head >>.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: <<_git_internals#_the_head>>

File: book/C-git-commands/1-git-commands.asc [hunk 13]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
 Ang `git merge` command ay unang ipinakilala sa << _ git_branching # _basic_branching >>.
 Kahit na ito ay ginagamit sa iba't ibang mga lugar sa aklat, napakakaunting mga pagkakaiba-iba ng 'merge` command - sa pangkalahatan lamang `git pagsamahin <branch>` na may pangalan ng solong sangay na nais mong pagsamahin.
 
-We covered how to do a squashed merge (where Git merges the work but pretends like it's just a new commit without recording the history of the branch you're merging in) at the very end of <<_distributed_git#_public_project>>.
-Tinakpan namin kung paano gumawa ng isang nakakulong na pagsasama (kung saan ang Git ay nagpahiwatig ng trabaho ngunit nagpapanggap na tulad ng isang bagong komit na hindi naitala ang kasaysayan ng sangay na pinagsasama mo) sa dulo ng << _ distributed_git # _public_project >>.
+Tinalakay namin kung paano gumawa ng isang squashed merge (kung saan ang Git ay nagpahiwatig na gumagana ang merge ngunit nagpapanggap lamang ito na tulad ng isang bagong commit na hindi naitala ang kasaysayan ng sangay na pinagsasama mo) sa pinakadulo ng << _ distributed_git # _public_project >>.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: <<_distributed_git#_public_project>>

File: book/C-git-commands/1-git-commands.asc [hunk 14]
```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
...
 
-We went over a lot about the merge process and command, including the `-Xignore-space-change` command and the `--abort` flag to abort a problem merge in <<_git_tools#_advanced_merging>>.
-Nagpunta kami ng maraming tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` command at ang` --abort` flag upang i-abort ang problema sa pagsasama sa << _ git_tools # _advanced_merging >>.
+Maramin din kaming tinalakay tungkol sa proseso ng pagsasama at utos, kabilang ang `-Xignore-space-change` na ustos at ang` --abort` na palatandaan upang i-abort ang problema sa pag-merge sa << _ git_tools # _advanced_merging >>.
```
Comment (severinolorillajr) [inline book/C-git-commands/1-git-commands.asc]: <<_git_tools#_advanced_merging>>

**Comment by severinolorillajr** [comment]
Hi @cherryloua , please be very careful with the tags,
for example <<_git_tools#_advanced_merging>>
don't remove the tags, or add spaces to it.
Just leave it as it is.
Thanks.

File: book/C-git-commands/1-git-commands.asc (hunk 1 above)
Comment (heywens) [reply book/C-git-commands/1-git-commands.asc]: Suggestions - please change the following words:
*  ``kautosang`` to ``kautusang``
*  ``kaligiran`` to ``environment`` 
hi @severinolorillajr, can we make this as one of the reserved words? or do we have to translate it? please advise.

**Comment by dranem** [comment]
change this one also << _ git_basics_chapter # _checking_status >> . Tip : View the original file so that you will have a guide for the original syntax.

File: book/C-git-commands/1-git-commands.asc (hunk 1 above)
Comment (mind-blast) [reply book/C-git-commands/1-git-commands.asc]: @heywens i think environment is better since it is a technical terms which really fits for git.

**Comment by cherryloua** [comment]
Thank you for the feedback guys.... I'm going to edit them now. Thanks again.

File: book/C-git-commands/1-git-commands.asc (hunk 6 above)
Comment (severinolorillajr) [reply book/C-git-commands/1-git-commands.asc]: Please update this as well. Thanks.

**Comment by severinolorillajr** [comment]
One peniding review, then we can merge this.

**Pull request merged**